web_timeline/static/src/css/web_timeline.css

```diff
@@ -0,0 +1,42 @@
+/* Button style */
+.openerp .oe_view_manager .oe_view_manager_switch .oe_vm_switch_timeline:after {
+    content: "N";
+}
+.timeline-navigation-zoom-in .ui-icon{
+  background: none !important;
+}
+
+.timeline-navigation-zoom-out .ui-icon{
+  background: none !important;
+}
+.timeline-navigation-move-left .ui-icon{
+  background: none !important;
+}
+.timeline-navigation-move-right .ui-icon{
+  background: none !important;
```

Comment:
Please avoid using `!important` if not absolutely required. It hardens the theming task later. Use a more specific CSS selector if you need to override some core behavior if possible please.